client/src/utils/router.constant.js

```diff
@@ -0,0 +1,10 @@
+export const HOME_PAGE = '/';
+export const BLOGS = '/blog';
+export const SIGNIN_PAGE = '/signin';
+export const SIGNUP_PAGE = '/signup';
+export const PROFILE = '/profile';
+export const CONTACTS = '/contact';
+export const BOOK = '/book';
+export const SERVICES = '/services';
+export const DASHBOARD = '/dashboard';
+export const DASHBOARD_APPOINTMENT = 'dashboard/appointment	';
```

Comment:
remove the spaces from the end of this constant so it will be '`dashboard/appointment'`